templates/Package_summary.ss

```diff
@@ -0,0 +1,4 @@
+<div class="package-summary">
+    <h5 class="package-summary__title">$Title</h5>
```

Comment:
We need to ensure that these variables are escaped - `$Title.XML` and `$Description.XML` - this casting is default in SS4 but needs to be enforced manually in SS3